Frameworks/QuartzCore/CALayer.mm

```diff
@@ -833,6 +831,8 @@ - (void)removeFromSuperlayer {
     [CATransaction _removeLayer:self];
 
     pSuper->priv->removeChild(self);
+
+    [pSuper setNeedsLayout];
     [self release];
```

Comment:
I have a repro and a test case in progress. will check it in separately.

---
In reply to: [90150570](https://github.com/Microsoft/WinObjC/pull/1471#discussion_r90150570) [](ancestors = 90150570)